### DISCUSSION TITLE
Put Image List In Drive Or Base

### DISCUSSION THREAD

**Discussion by w3bsite** [deta/cloud-docs#361] [Help] [answered]
hi what is the correct aproach for storing a list of images that are related to some posts in drive?
should I put them along those post as a base64 in base ?
or upload them in the drive,
I was going with the drive but Im not sure how to fetch the images list, it didnt feel right, but I was going to fetch the list of images names first and then fetch each image by it's name and push it to an array and bring it back from micro , but as I said it did not seemed right to me so I wanted to ask for the right aproach, thank you

**Comment by abdelhai** [comment, marked as answer]
store the actual files in drive and their names/metadata in base. then you can query base for whichever files you want then retrieve them from drive by name.

**Reply by w3bsite** [reply]
thanks for the answer, in your opinion is it fine to do this for a product list/shop website, there is going to be many products to show the user with their picture, is drive the right tool for this job gennerally?

**Reply by abdelhai** [reply]
this is the correct implantation, even large scale apps separate the actual storage and file meta data. so, yes, i feel drive is the right tool for that
